ci/builders/mac_unopt.json

```diff
@@ -65,7 +65,7 @@
                     "runtime_versions": [
                         "ios-16-0_14a5294e"
                     ],
```

Comment:
I believe the simulator runtimes need to be updated as well:
https://github.com/flutter/flutter/issues/125216#issuecomment-1516657893

Reply:
Added and the CI is happy now.